### PR TITLE
Remove alert modal from localForage error case

### LINKED ISSUE #1626: Navigating to <Siteroot>/admin/ page leads to a system error popup
**Describe the bug**
So I just setup netlify-cms on my site and I saw that when I tried to navigate to <Siteroot>/admin/ I get an error popup below.
screen shot 2018-08-17 at 9 33 05 am

After clicking Ok, the admin page loads like normal and I'm able to interact with my collections, but I was just wondering what was causing this, since it's pretty alarming to our other users that modify the CMS content.

Thanks!

**Applicable Versions:**

 - Netlify CMS version: 1.7.0
 - Git provider: Github
 - OS: Mac OSX 10.13.2
 - Browser version Chrome Version 67.0.3396.99

### PR DESCRIPTION
**Pull request by tonyrzhang**
**Summary**

Fixes #1626
Remove alert modal from localForage error case

**Test plan**
yarn test

### CODE REVIEW

**Comment by tonyrzhang** [comment]
@tech4him1

LMK if this works for you.

**Comment by verythorough** [comment]
Deploy preview for *netlify-cms-www* ready!

Built with commit 8491baad90880855cf184892f11f00b2b66f731a

https://deploy-preview-1676--netlify-cms-www.netlify.com

**Comment by verythorough** [comment]
Deploy preview for *cms-demo* ready!

Built with commit 8491baad90880855cf184892f11f00b2b66f731a

https://deploy-preview-1676--cms-demo.netlify.com

File: packages/netlify-cms-lib-util/src/localForage.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ function localForageTest() {
     })
     .catch(err => {
       if (err.code === 22) {
-        const message = `Unable to set localStorage key. Quota exceeded! Full disk?`;
-        return alert(`${message}\n\n${err}`);
+        const message = 'Unable to set localStorage key. Quota exceeded! Full disk?';
+        console.log(message);
```
Comment (tech4him1) [inline packages/netlify-cms-lib-util/src/localForage.js]: Let's do a `console.warn` here. Other than that, LGTM!

**Comment by tonyrzhang** [comment]
Cool thanks! And I assume you guys will have to merge it as well?

**Comment by tech4him1** [comment]
Yes, once another maintainer reviews they will merge it.

**Pull request merged**